### DISCUSSION TITLE
Can we make a release please?

### DISCUSSION THREAD

**Discussion by SupermanAdam007** [rabbitmq/messaging-topology-operator#498] [General]
Hi, can we make a release please? I really need this feature: https://github.com/rabbitmq/messaging-topology-operator/commit/60556f8fbe98f76768bad2f79c8885dc44f1cb1c :)

Thank you!

**Comment by ChunyiLyu** [comment]
You can find the latest one here: https://github.com/rabbitmq/messaging-topology-operator/releases/tag/v1.10.0 :)

**Reply by SupermanAdam007** [reply]
❤️ thanks!
